tests/Unit/Service/AntiSpamServiceTest.php

```diff
@@ -34,6 +34,8 @@ class AntiSpamServiceTest extends TestCase {
 	private SmtpClientFactory|MockObject $smtpClientFactory;
 	private MockObject|ImapMessageMapper $imapMessageMapper;
 	private LoggerInterface|MockObject $logger;
+	private MockObject|IMailTransmission $transmission;
```

Comment:
```suggestion
	private IMailTransmission|MockObject $transmission;
```

for consistency and readability

Reply:
not a blocker. we can merge regardless